core/modules/system/system.menu.inc

```diff
@@ -107,7 +107,7 @@ function system_menu_block_form($config) {
     '#type' => 'checkbox',
     '#title' => t('Display menu toggle button on small screens'),
     '#default_value' => $config['toggle'],
-    '#description' => t('A menu toggle button, commonly known as a hamburger icon, allows the menu to be hidden initially on devices with small screens, but then easily expanded.'),
+    '#description' => t('On devices with small screens, this option reduces the menu to a toggle button, commonly known as <a href="@url" target="_blank"><em>hamburger button</em></a>. Clicking the icon reveals or hides the menu, saving screen space.', array('@url' => 'https://en.wikipedia.org/wiki/Hamburger_button')),
```

Comment:
In this case, the URL should be a part of the string, so the link itself can go to a localized version of the Wikipedia page for Hamburger Icon.